src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,52 @@
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  isTodoSelected: boolean;
+  onTodoSelect: (todo: Todo) => void;
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  isTodoSelected,
+  onTodoSelect,
+}) => {
+  return (
+    <tr data-cy="todo" className="">
+      <td className="is-vcentered">{todo.id}</td>
+      <td className="is-vcentered">
+        {todo.completed && (
+          <span className="icon" data-cy="iconCompleted">
+            <i className="fas fa-check" />
+          </span>
+        )}
+      </td>
+      <td className="is-vcentered is-expanded">
+        <p className={cn({
+          'has-text-success': todo.completed,
+          'has-text-danger': !todo.completed,
```

Comment:
one classname can serve as a default one. It will be overwritten if matches other condition